src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py

```diff
@@ -1126,3 +1134,32 @@
   - name: Get the private link resource for a flexible server using --ids parameter.
     text: az postgres flexible-server private-link-resource show --ids /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/testGroup/providers/Microsoft.DBforPostgreSQL/flexibleServers/testServer
 """
+
+helps['postgres flexible-server fabric-mirroring'] = """
+type: group
+short-summary: Bring your PostgreSQL data into Microsoft Fabric. Mirroring allows you to create a replica of your data in OneLake which can be used for all your analytical needs.
+"""
+
+helps['postgres flexible-server fabric-mirroring start'] = """
+type: command
+short-summary: Enable bringing your PostgreSQL data into Microsoft Fabric.
+examples:
+  - name: Enable bringing your PostgreSQL data into Microsoft Fabric.
+    text: az postgres flexible-server fabric-mirroring start -g testgroup -s testsvr --database-names testdb
```

Comment:
nit: maybe you can share example where there are multiple database names

Reply:
We show that example in update-databases